docs/docs/tutorials/fundamentals/faststore.md

```diff
@@ -0,0 +1,33 @@
+---
+id: 1
+title: Faststore
+sidebar_label: What is Faststore
+---
+
+*[Faststore](https://github.com/vtex/faststore) is a  collection of **open-source** libraries consisting of a ecommerce-focused SDK package, a React UI components library and a GraphQL API to connect to your ecommerce provider.*
+
+---
+
+## What is Faststore 
+
+[Faststore](https://github.com/vtex/faststore) is a light yet powerful **set of libraries** made for developers who want to explore their knowledge and creativity building comprehensive and unique ecommerce solutions. It supports integrations with headless ecommerce platforms, allows meaningful analytics capabilities, and gives developers back the freedom of customization.
+
+## What is included
+
+That is possible because Faststore was cleverly split into **a UI, SDK and API packages.** The UI package is a **CSS framework-agnostic** library that crafts the behavior of React components into data attributes. The SDK is an extensible state management library that handles all meaningful states an e-commerce store might have, such as:
```

Comment:
```suggestion
That is possible because FastStore was cleverly split into **a UI, SDK and API packages.** The UI package is a **CSS framework-agnostic** library that crafts the behavior of React components into data attributes. The SDK is an extensible state management library that handles all meaningful states an ecommerce store might have, such as:
```